CSharpSource/Source/Microsoft.Xbox.Services.UWP.CSharp/Social/Manager/SocialManager.cs

```diff
@@ -171,9 +186,10 @@ public IList<SocialEvent> DoWork()
                         group.Refresh();
                     }
                 }
-                foreach (XboxLiveUser user in m_localUsers)
+
+                foreach (IntPtr userPtr in m_localUsers.Keys)
```

Comment:
Can't you just iterate over the values instead of the keys? You save the extra lookup inside the loop